### PR TITLE
update LSP0 interface ID + add ClaimOwnership interface ID

### PR DESCRIPTION
**Pull request by CJ42**

### CODE REVIEW

File: docs/standards/smart-contracts/interface-ids.md [hunk 1]
```diff
@@ -19,9 +19,10 @@ _Interface IDs are not the most secure way to ensure that a contract implements
 | **ERC725Y**                       | `0x714df77c` | General Data key-value store.                                         |
 | **LSP1UniversalReceiver**         | `0x6bb56a14` | Universal Receiver entry function.                                    |
 | **ERC1271**                       | `0x1626ba7e` | Standard Signature Validation Method for Contracts.                   |
-| **LSP0ERC725Account**             | `0x481e0fe8` | Account that represent an identity on-chain                           |
+| **LSP0ERC725Account**             | `0x9a3bfe88` | Account that represent an identity on-chain                           |
```
Comment (Hugoo) [inline docs/standards/smart-contracts/interface-ids.md:22]: where can i verify this value?

Comment (CJ42) [reply docs/standards/smart-contracts/interface-ids.md:22]: @Hugoo 
https://github.com/lukso-network/lsp-smart-contracts/blob/7b353b17bc16d2fc64f4d57955eba016bb79c4bb/constants.ts#L20

We also calculate them + test them to ensure that they are always correct.
If we enter them incorrectly, the CI will fail.

https://github.com/lukso-network/lsp-smart-contracts/blob/7b353b17bc16d2fc64f4d57955eba016bb79c4bb/tests/Helpers/ERC165Interfaces.test.ts#L31-L34

https://github.com/lukso-network/lsp-smart-contracts/blob/develop/contracts/Helpers/ERC165Interfaces.sol#L39-L56

**Review by Hugoo — APPROVED** [review]

**Comment by Hugoo** [comment]
@CJ42 maybe you can also add on this page a reference to this:
<https://github.com/lukso-network/lsp-smart-contracts/blob/7b353b17bc16d2fc64f4d57955eba016bb79c4bb/constants.ts#L9>

in a callout bellow the table:
image


So readers can find where these values are set ?

**Comment by frozeman** [comment]
I added a reference to the constants files, but i think we should rethink this file

**Review by YamenMerhi — APPROVED** [review]

**Review by frozeman — APPROVED** [review]

**Pull request merged**